### PR TITLE
Basque (eu) translation

### PR DESCRIPTION
**Pull request by erral**
Add basque (eu) translation of strings.xml

### CODE REVIEW

File: app/src/main/res/values-eu/strings.xml [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
...e forma anónima y descentralizada en los dispositivos de los USUARIOS durante un periodo no superior a 14 días;\n\ny c) la información ofrecida al USUARIO sobre el posible riesgo de contagio, sin que en ningún momento se refieran datos personales de ningún tipo.\n\nEl USUARIO puede informar voluntariamente a la aplicación de un resultado positivo en sus pruebas de COVID-19 mediante el código de confirmación de un solo uso facilitado por las autoridades sanitarias. La validez de este código será cotejada por las autoridades sanitarias para asegurar el correcto funcionamiento de Radar COVID. El USUARIO informará de los resultados de sus pruebas y se le solicitará el consentimiento expreso e inequívoco para compartir las claves generadas diariamente en su dispositivo, y correspondientes a los últimos 14 días. Estas claves son comunicadas a un servidor que las pondrá a disposición del conjunto de aplicaciones Radar COVID para su descarga. Las claves comunicadas no guardan relación alguna con la identificación del dispositivo o el USUARIO.\n\nNo se producirá ninguna discriminación a los potenciales pacientes que requieran servicios sanitarios y no hayan utilizado la aplicación.</string>
```
Comment (Loksly) [inline app/src/main/res/values-eu/strings.xml:47]: te ha faltado alguna parte, ¿no?

**Review by Loksly — COMMENTED** [review]
buena aportación

**Comment by erral** [comment]
Sí. sí, falta la parte del texto legal. Por esto tiene el prefijo [WIP] de Work in Progress. He hecho el PR para que se viera que alguien estaba haciendo ya la traducción y no duplicar esfuerzos. Espero tener eso preparado hoy o mañana,

**Comment by erral** [comment]
The translation is ready.

**Comment by siempredelao** [comment]
Are translations handled through the different `strings.xml` files? In the onboarding screen, I see a language selector but I only see `strings.xml` for the default language (Spanish), so I don't know if translations are handled through the resource files or through another mechanism. Could anybody please clarify this?

**Comment by erral** [comment]
The language configuration settings are loaded through the backend-configuration server: 

https://github.com/RadarCOVID/radar-covid-backend-configuration-server#endpoints

But the example response texts published in github are only in spanish:

https://github.com/RadarCOVID/radar-covid-backend-configuration-server/blob/master/responses/response-texts.json

**Comment by erral** [comment]
According to this comment, the translation of the app comes from a Contentful CMS, so this PR unnecessary.

**Pull request closed without merging**